ios/AdvancedTextInputMaskDecoratorView.swift

```diff
@@ -140,41 +145,72 @@ class MaskedTextInputDecoratorView: UIView {
       return
     }
 
-    backedTextInputView.allText = result.formattedText.string
-    maskInputListener?.notifyOnMaskedTextChangedListeners(forTextInput: backedTextInputView as! UITextField, result: result)
+    textView.allText = result.formattedText.string
+    maskInputListener?.notifyOnMaskedTextChangedListeners(forTextInput: textView, result: result)
   }
 
-  override func didMoveToWindow() {
-    super.didMoveToWindow()
+  private func findFirstTextField(in view: UIView) -> UITextField? {
+    // Loop through each subview
+    for subview in view.subviews {
+      // If the subview is a UITextField, return it
+      if let textField = subview as? UITextField {
+        return textField
+      }
+    }
+    // If no UITextField is found, return nil
+    return nil
+  }
 
-    var previousSibling: UIView?
+  private func findTextFieldFabric() {
+    if let parent = superview?.superview {
+      for i in 1 ..< parent.subviews.count {
+        if parent.subviews[i] == superview {
+          textView = findFirstTextField(in: parent.subviews[i - 1])
+          break
+        }
+      }
+    }
+  }
+
+  private func findTextFieldPaper() {
     if let parent = superview {
       for i in 1 ..< parent.subviews.count {
         if parent.subviews[i] == self {
-          previousSibling = parent.subviews[i - 1]
+          textView = findFirstTextField(in: parent.subviews[i - 1])
           break
         }
       }
     }
+  }
 
-    if let previousSibling = previousSibling as? RCTBaseTextInputView {
-      textView = previousSibling
-      guard let textView = textView?.backedTextInputView as? RCTUITextField else { return }
-      maskInputListener = MaskedTextInputListener(
-        primaryFormat: primaryMaskFormat as String,
-        autocomplete: autocomplete,
-        autocompleteOnFocus: autocompleteOnFocus,
-        autoskip: autoSkip,
-        rightToLeft: isRTL,
-        affineFormats: affinityFormat,
-        affinityCalculationStrategy: AffinityCalculationStrategy.forNumber(number: affinityCalculationStrategy), customNotations: (customNotations as? [[String: Any]])?.map { $0.toNotation() } ?? [],
-        onMaskedTextChangedCallback: { input, value, _, _ in
-          self.onAdvancedMaskTextChangedCallback(value, input.allText)
-        },
-        allowSuggestions: allowSuggestions
-      )
-      textView.delegate = maskInputListener
-      updateTextWithoutNotification(text: defaultValue as String)
-    }
+  private func finTextField() {
```

Comment:
fin?) 🇫🇮

Reply:
Better to add spell check to prevent such issues in future 🙂 
I found a way how to run `cspell` for entire project, you know where you can take a reference 🙂